src/containers/Chat/ChatMessages/ChatMessages.tsx

```diff
@@ -221,6 +221,11 @@ export const ChatMessages: React.SFC<ChatMessagesProps> = ({
 
   // HOOKS ESTABLISHED ABOVE
 
+  // check if the search API results nothing for a particular contact ID and redirect to chat
+  if (contactId && data && data.search.length === 0) {
```

Comment:
Let's merge these two conditions into one.
if (contactId && data) {
  if (data.search.length === 0 || data.search[0].contact.status === 'BLOCKED') {	
    return <Redirect to="/chat" />;
  }
}

Reply:
Sure